app/models/user.rb

```diff
@@ -3,7 +3,11 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  has_and_belongs_to_many :projects
+
+  has_many :projects_users, class_name: "ProjectsUsers",
```

Comment:
You don't need to specify the class_name, if rails can deduce it by the relation name.